src/lib.rs

```diff
@@ -107,7 +107,6 @@
 //! - Only support multicast, not unicast send/recv.
 //! - Only support 32-bit or bigger platforms, not 16-bit platforms.
 
-#![forbid(unsafe_code)]
 #![allow(clippy::single_component_path_imports)]
```

Comment:
Following my comment above, I suggest enabling these lints:

```rust
#![warn(unsafe_op_in_unsafe_fn)]
#![warn(clippy::undocumented_unsafe_blocks)]
#![warn(clippy::multiple_unsafe_ops_per_block)]
```

Reply:
We cannot remove `#![forbid(unsafe_code)]` just because one dependency lib changed its API to be `unsafe`. Sorry.